modyn/models/dlrm/dlrm.py

```diff
@@ -111,7 +111,8 @@ def reorder_categorical_input(self, cat_input: torch.Tensor) -> torch.Tensor:
         order = self._embedding_ordering.expand(dim0, -1)
         return torch.gather(cat_input, 1, order)
 
-    def forward(self, data: torch.Tensor) -> torch.Tensor:
+    # pylint: disable-next=unused-argument
```

Comment:
there is no `del sample_ids` in this file. Sorry I didn't get it?

Reply:
My point is that you add a pylint nolint to supress a warning that you dont use a parameter. Instead, you should add a line del sample_ids since that avoids the nolint